client/src/components/App.js

```diff
@@ -3,6 +3,7 @@ import { BrowserRouter, Route } from 'react-router-dom';
 import { connect } from 'react-redux';
 import * as actions from '../actions';
 import { Switch } from 'react-router';
+import styles from '../styles/global.css';
 
```

Comment:
We have to import the global CSS so that our root component and its nested components can use the CSS.